Library/Homebrew/keg_relocate.rb

```diff
@@ -78,7 +78,7 @@ def text_files
       output, _status = Open3.capture2("/usr/bin/xargs -0 /usr/bin/file --no-dereference --brief",
                                        stdin_data: files.join("\0"))
       output.each_line.with_index do |line, i|
-        next unless line.include?("text")
+        next unless line.include?("text") || files[i].text_executable?
```

Comment:
Make these two separate `next` for readability and "Mike can't parse `unless ... ||`"